src/resources/js/store/localisation.js

```diff
@@ -6,11 +6,32 @@ export const state = {
 
 export const getters = {
     isInitialised: state => Object.keys(state.i18n).length > 0,
-    __: (state, getters, rootState, rootGetters) => (key) => {
-        const lang = rootGetters['preferences/lang'];
-        return state.i18n[lang]
+    __: (state, getters, rootState, rootGetters) => (key, params) => {
+        const lang = rootGetters['preferences/lang']
+            || ((rootState.preferences && rootState.preferences.global)
+                ? rootState.preferences.global.lang : null);
```

Comment:
I'm not sure that I understand why this ternary is needed

Reply:
I had a case (during login if I remember correctly) that, for some reason, the getter wasn't correctly returning the language but the rootstate was (which prevented the correct language from showing after navigating to the framework using a non-english language) I'll check if this is still needed

Reply:
confirming the _fix_ is still required for translations to work after pageload